lldb/packages/Python/lldbsuite/test/tools/lldb-dap/dap_server.py

```diff
@@ -895,6 +906,32 @@ def request_setFunctionBreakpoints(self, names, condition=None, hitCondition=Non
         }
         return self.send_recv(command_dict)
 
+    def request_dataBreakpointInfo(self, variablesReference, name):
+        args_dict = {"variablesReference": variablesReference, "name": name}
+        command_dict = {
+            "command": "dataBreakpointInfo",
+            "type": "request",
+            "arguments": args_dict,
+        }
+        return self.send_recv(command_dict)
+
+    def request_setDataBreakpoint(self, dataBreakpoints):
+        """dataBreakpoints is a list of dictionary with following fields:
```

Comment:
`list of dictionary` sounds very weird. Did you mean something else?

Reply:
`dataBreakpoints` is a list of the dataBreakpoint which has following format:
```
{
    dataId: (address in hex)/(size in bytes)
    accessType: read/write/readWrite
    [condition]: string
    [hitCondition]: string
}
```